### PR TITLE
Add missing backend translations for SK language

### PR DESCRIPTION
**Pull request by Samuell1**

### CODE REVIEW

File: modules/backend/lang/sk/lang.php [hunk 1]
```diff
@@ -15,6 +16,11 @@
     ],
     'page' => [
         'untitled' => 'Bez názvu',
+        '404' => [
+            'label'     => 'Stránka nenájdená',
+            'help'      => "Hľadali sme a hľadali, ale požadovanú adresu URL jednoducho nebolo možné nájsť. Možno ste hľadali niečo iné?",
+            'back_link' => 'Späť na predchodzú stránku',
```
Comment (bennothommo) [inline modules/backend/lang/sk/lang.php]: @Samuell1 Would you be able to remove the extra spaces before the arrow indicators? The rest of the document seems to be using the `space arrow space` format.

Comment (Samuell1) [reply modules/backend/lang/sk/lang.php]: I can but this was copied from en lang.php file

Comment (LukeTowers) [reply modules/backend/lang/sk/lang.php]: I'm fine with the aligned => format, eventually I'd like it everywhere in the lang files

Comment (Samuell1) [reply modules/backend/lang/sk/lang.php]: Anything needed? @LukeTowers

**Pull request merged**